Set additional variables (boot_targets, devtype, devnum, bootpart) to
emulate config_distro_bootcmd.h so that the debian-installer
bootscript will work.

Index: u-boot/include/configs/mx6_cubox-i.h
===================================================================
--- u-boot.orig/include/configs/mx6_cubox-i.h
+++ u-boot/include/configs/mx6_cubox-i.h
@@ -286,6 +286,10 @@
 
 #define CONFIG_BOOTCOMMAND \
 	   "mmc dev ${mmcdev}; if mmc rescan; then " \
+               "setenv boot_targets mmc ; " \
+               "setenv devtype mmc ; " \
+               "setenv devnum ${mmcdev} ; " \
+               "setenv bootpart ${mmcpart} ; " \
                "for prefix in ${boot_prefixes}; do " \
 		   "setenv file_prefix ${prefix}; " \
 		   "run autodetectfdt; " \
